net-misc/tigervnc/tigervnc-1.12.0-r1.ebuild

```diff
@@ -171,10 +169,20 @@ src_install() {
 			rm -v "${ED}"/usr/$(get_libdir)/xorg/modules/extensions/libvnc.la || die
 		fi
 
-		newconfd "${FILESDIR}"/${PN}.confd ${PN}
-		newinitd "${FILESDIR}"/${PN}.initd ${PN}
+		newconfd "${FILESDIR}"/${PN}-${PV}.confd ${PN}
+		newinitd "${FILESDIR}"/${PN}-${PV}.initd ${PN}
+
+		# this file is here to document what I tested and what works
+		# TODO: remove it when done
+		if [ -f "${ED}"/etc/tigervnc/vncserver-config-defaults ]; then
+			cat "${FILESDIR}"/vncserver-config-defaults >> "${ED}"/etc/tigervnc/vncserver-config-defaults || die
+		fi
 
 		systemd_douserunit unix/vncserver/vncserver@.service
+
+		# comment out pam_selinux.so, the server does not start if missing
+		# TODO: add an USE flag maybe?
+		sed -i -e '/pam_selinux/s/^/#/' "${ED}"/etc/pam.d/tigervnc
```

Comment:
[|| die on external commands which may fail](https://devmanual.gentoo.org/ebuild-writing/error-handling/#the-die-function)

Reply:
.. and pop in a bug reference too